README.md

```diff
@@ -91,6 +100,15 @@ You can find test assembly programs in the `tests/` directory. These files provi
   - `SBC` – Subtract one register value from another with carry.
   - `SBRC` – Skip next instruction if bit in register is cleared.
   - `SBRS` – Skip next instruction if bit in register is set.
+  - `SEC` - Set the Carry flag.
+  - `SEH` - Set the Half Carry flag.
+  - `SEI` - Set the Global Interrupt flag.
+  - `SEN` - Set the Negative flag.
+  - `SER` - Sets all bits in a regitser.
+  - `SES` - Set Signed flag.
+  - `SET` - Set T flag.
+  - `SEV` - Set Overflow flag.
+  - `SEZ` - Set Zero flag.
```

Comment:
The dash after the instruction name is not the same dash as everywhere else. You are using "-" instead of "–", which is a long dash. Can you please change it so it matches?